typescript/infra/config/environments/mainnet2/agent.ts

```diff
@@ -48,33 +50,75 @@ const contextBase = {
   },
 } as const;
 
+const bscNautilusWarpRoutes: Array<{ router: string }> = [
+  // ZBC
+  {
+    router: '0xC27980812E2E66491FD457D488509b7E04144b98',
+  },
+  // ETH
+  {
+    router: '0x2a6822dc5639b3fe70de6b65b9ff872e554162fa',
+  },
+  // USDC
+  {
+    router: '0x6937a62f93a56D2AE9392Fa1649b830ca37F3ea4',
+  },
+  // BTC
+  {
+    router: '0xB3545006A532E8C23ebC4e33d5ab2232Cafc35Ad',
+  },
+  // USDT
+  {
+    router: '0xb7d36720a16A1F9Cfc1f7910Ac49f03965401a36',
+  },
+  // POSE
+  {
+    router: '0x97a2D58d30A2c838946194494207F7Cf50c25815',
+  },
+];
+
 const gasPaymentEnforcement: GasPaymentEnforcementConfig[] = [
   {
     type: GasPaymentEnforcementPolicyType.None,
     // To continue relaying interchain query callbacks, we whitelist
     // all messages between interchain query routers.
     // This whitelist will become more strict with
     // https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/1605
-    matchingList: interchainQueriesMatchingList,
+    matchingList: [
+      ...interchainQueriesMatchingList,
+      {
```

Comment:
we can remove all these warp routes & matching list changes if we want - or we can keep them in, and just remove when we deploy mainnet3, either way

Reply:
we havent touched mainnet2 on v3 so I will leave this